docs/monitoring-guidelines.md

```diff
@@ -9,7 +9,12 @@ The KubeVirt Users should have the same experience when searching for a node, co
 
 **Naming requirements:**
 1. Check if a similar Kubernetes metric, for node, container or pod, exists and try to align to it.
-2. KubeVirt metric for a running VM should have a `kubevirt_vmi_` prefix
+2. KubeVirt metrics prefixes:
+   1.  Running VM metrics should have a `kubevirt_vmi_` prefix
+   2.  HCO operator metrics should have a `kubevirt_hco_` prefix
+   3.  Network operator metrics should have a `kubevirt_network_` prefix
```

Comment:
Not sure if this one is fitting. Our network metrics that we can collect on kubevirt/kubevirt are related to VMI only. The other related network project we ship under HCO are not under the kubevirt/ GitHub organization. With this, we would not prefix metrics on e.g. kubernetes-nmstate with kubevirt_.

Reply:
This should be a prefix for network operator metrics that report on the operator itself. Not VMI network metrics.

Reply:
Oh, I see. Looks good to me then, thanks for the clarification